zephyr-env.cmd

```diff
@@ -4,3 +4,6 @@ set ZEPHYR_BASE=%~dp0
 if exist "%userprofile%\zephyrrc.cmd" (
```

Comment:
Hmm, this file looks out of place here.

Reply:
This has been fixed in the latest pull request: https://github.com/zephyrproject-rtos/zephyr/pull/23478